packages/contracts-bedrock/scripts/print-addresses.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Default directory base path
+BASE_DIR="./deployments"
+
+# Directory name from the argument
+DIR_NAME=$1
+
+# Check for the --sdk flag
+SDK_MODE=false
+if [[ "$2" == "--sdk" ]]; then
+    SDK_MODE=true
+fi
+
```

Comment:
Consider defining a help function and then check that `$1` is defined and help if not, also if `$DIR` doesn't exist

Reply:
Done